### PR TITLE
Display Status to vue Comm lists

### PR DESCRIPTION
**Pull request by leobcgov**
# Fixes or Implements [AB#8218](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8218)

* [X] Enhancement
* [ ] Bug
* [ ] Documentation

## Description
Diplay Status to vue Comm lists
image

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [X] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

YES

### Browsers Tested

* [ ] Chrome
* [ ] Safari
* [ ] Edge
* [X] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.  

</br>
</br>
</br>
___
  
## Reviewer Guidelines

A reviewer should examine the PR if the Status above indicates the PR is **Ready**.
Once ready, the reviewer should read the information below as well as supporting information like the Azure ticket to ensure they have a good understanding of the request.

The reviewer should then:

1. Is the PR template complete and accurate?

    Confirm by reading all sections below.

1. Confirm the merge branch

    A normal feature/bug branch should merge into the Dev branch.  Exceptions for hotfix and sprint merges do occur and the target would then be Master.

1. Do the code commits have an appropriate comment and include linkage to the azure ticket by including a reference to "AB#nnnn"?

1. Review the Code Changes

    1. Do the changes match the description?

        The code commits should reflect the documented description and nothing more.  If additional changes are included the reviewer should call these out and request that the developer document the changes in Azure and in the description or remove the changed code.

    1. useful tips

        * Any obvious errors or omissions?
        * Does the code match the style of other similar work?
        * Is the code easy to understand?
        * Is the code well documented?
        * DB Migrations are appropriate?

If the Answer to any of the questions above is No, then a request for change should be the result of the review.

### CODE REVIEW

**Review by sslaws — APPROVED** [review]

File: Apps/AdminWebClient/src/ClientApp/src/models/adminCommunication.ts [hunk 1]
```diff
@@ -40,6 +45,9 @@ export default interface Communication {
     // Gets or sets the communication status code: new or processed
     communicationStatusCode: string;
 
+    // Gets or sets the communication publishing status code: Draft or Publish
+    publishingStatusCode: string;
```
Comment (FuriousLlama) [inline Apps/AdminWebClient/src/ClientApp/src/models/adminCommunication.ts:49]: should this be an enum too?

Comment (leobcgov) [reply Apps/AdminWebClient/src/ClientApp/src/models/adminCommunication.ts:49]: no, I don't think so as the communicationStatusCode above is also a string right?

Comment (FuriousLlama) [reply Apps/AdminWebClient/src/ClientApp/src/models/adminCommunication.ts:49]: if both can be changed that might be even better. That's -if-

Comment (leobcgov) [reply Apps/AdminWebClient/src/ClientApp/src/models/adminCommunication.ts:49]: Sounds good, I will look at our other areas to see if we can do this easily here, if not I will create a ticket to capture this refactoring later then.

Comment (leobcgov) [reply Apps/AdminWebClient/src/ClientApp/src/models/adminCommunication.ts]: As we don't introduce the new Publishing Status, I prefer to defer this refactoring to another task then.

**Review by FuriousLlama — APPROVED** [review]

**Review by tiagograf-AOT — APPROVED** [review]

**Pull request merged**